zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -76,6 +83,50 @@ public MultipleRegexesOptionsPanel(Dialog owner, RegexesTableModel model) {
 
         this.owner = owner;
         getTable().setSortOrder(0, SortOrder.ASCENDING);
+        JButton importButton =
+                new JButton(Constant.messages.getString("context.include.import.label"));
+        importButton.addActionListener(e -> importButtonClicked(e));
+        addButton(importButton);
+    }
+
+    public void importButtonClicked(ActionEvent event) {
+        File file = selectImportFile();
+        if (file != null) {
+            List<String> regexes = getRegexes();
```

Comment:
This is not a copy, this will lead to partial imports in case of failure (which causes other issues as the changes to the model are not notified to the view).

Reply:
created local copy to allow duplication checking